inc/Mailer.class.php

```diff
@@ -715,6 +716,14 @@ public function send() {
             if(trim($data['headers'])) {
                 $headers .= MAILHEADER_EOL.trim($data['headers']);
             }
+            if(!function_exists('mail')){
+                $emsg = $lang['email_fail'] . $subject;
+                error_log($emsg);
+                if(in_array('admin', $USERINFO['grps'])){
+                    msg($emsg);
+                }
+                // return false;
```

Comment:
Why don't you just `return false;` here? I don't think `$evt->advise_after()` needs to be triggered if failed - correct me if I am wrong.

Reply:
Taking into consideration both of your suggestions, I assume this is what would work?
```
            if(!function_exists('mail')){
                $emsg = $lang['email_fail'] . $subject;
                error_log($emsg);
                msg($emsg,2,"","", MSG_MANAGERS_ONLY);               
                return false;
            }
```
I commented out `return false`  because I wasn't sure of the consequences of returning from here and was hoping someone would pick up on that.  As for the `MSG_MANAGERS_ONLY`, I had never looked at the code for `msg`. Thanks.